src/py/flwr/server/app.py

```diff
@@ -500,6 +512,12 @@ def _run_fleet_api_rest(
     # See: https://www.starlette.io/applications/#accessing-the-app-instance
     fast_api_app.state.STATE_FACTORY = state_factory
 
+    host, port_str = address.split(":")
+    port = int(port_str)
+
+    if not isfile(ssl_keyfile) or not isfile(ssl_certfile):
+        raise FileNotFoundError("One of SSL Keyfile or Certfile is not an actual file.")
```

Comment:
Let's split this into two if statements and tell the user exactly which path does not point to a file

Reply:
@danieljanes I rewrote it.